src/chromeDebugAdapter.ts

```diff
@@ -352,10 +352,8 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
         this.events.emitStepStarted('LaunchTarget.LaunchExe');
         const platform = coreUtils.getPlatform();
         if (platform === coreUtils.Platform.Windows && shouldLaunchUnelevated) {
-            const semaphoreFile = path.join(os.tmpdir(), 'launchedUnelevatedChromeProcess.id');
-            if (fs.existsSync(semaphoreFile)) { // remove the previous semaphoreFile if it exists.
-                fs.unlinkSync(semaphoreFile);
-            }
+            const semaphoreFile = path.join(os.tmpdir(), utils.generateRandomString(8) + '.id');
+
```

Comment:
How do we handle failures in the launchUnelevated.js file? Do we get any telemetry or logging?